### ISSUE TITLE
QuickFolders overwrites Automatic Bcc set in Thunderbird 

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#183]
Hi.  I'm not sure if you're aware, but your app appears to block the functionality of Thunderbird's auto-BCC setting.

In other words, Thunderbird allows a user to automatically bcc themself in every message sent, whether it is an original message, a forward, or a reply.  But with your app running, the auto BCC does not work when replying.

Can you investigate, and if confirmed, fix this?

**Comment by RealRaven2000** [comment]
It works for me. Here are the steps I took.
1) went to account settings, then clicked on Copies  & folders
2) activated the box "Bcc these email addresses:" and entered one
![image](https://user-images.githubusercontent.com/6314674/125775903-ef24491b-6266-48f8-8337-7623b1d6e7b0.png)
3) clicking on compose:
![image](https://user-images.githubusercontent.com/6314674/125776020-9fefe8d2-075b-4c60-bdef-5e078fc164b5.png)


- Can you check your error log? (you can find it in tools / developer tools / error console. Best to reset it before you click compose to leave out unnecessary messages.
- Have you got any "Tab-specific properties" defined in the folder you write your emails from?
![image](https://user-images.githubusercontent.com/6314674/125776189-d3873d31-e30a-46cb-8f52-e11a39bb8d02.png)
- Are you running any other Add-ons that may set address fields in composer?

**Comment by fredstluka** [comment]
Axel,

> Hi. I'm not sure if you're aware, but your app appears to block the 
> functionality of Thunderbird's auto-BCC setting.
>
> In other words, Thunderbird allows a user to automatically bcc 
> themself in every message sent, whether it is an original message, a 
> forward, or a reply. But with your app running, the auto BCC does not 
> work when replying.
>
> Can you investigate, and if confirmed, fix this?
>

I'm NOT seeing this problem.  I've had Thunderbird configured
for over 10 years to BCC me on every email I send.  And I've
used QuickFolders for a couple of years.  Never had any
problem with the BCC.

BTW, I'm not sure who reported this bug.  It looks to me like you
did, so maybe someone else sent you mail about it or something
and you cut/pasted into GitHub for them?

--Fred
------------------------------------------------------------------------
Fred Stluka -- http://bristle.com -- Glad to be of service!
Open Source: Without walls and fences, we need no Windows or Gates.
------------------------------------------------------------------------

**Comment by RealRaven2000** [comment]
> I'm NOT seeing this problem.  I've had Thunderbird configured for over 10 years to BCC me on every email I send.  And I've used QuickFolders for a couple of years.  Never had any problem with the BCC. BTW, I'm not sure who reported this bug.  It looks to me like you did, so maybe someone else sent you mail about it or something and you cut/pasted into GitHub for them?

yes. The user raised it almost 2 days ago and I raised the issue for him. I sent him a link to the Github and haven't heard back grom him yet...
